client/packages/common/src/ui/icons/Help.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+export const HelpIcon = (props: SvgIconProps): JSX.Element => {
+  return (
+    <SvgIcon
+      {...props}
+      style={{
+        fill: 'currentColor',
+        strokeWidth: 2,
```

Comment:
This is *super* picky, but the question mark appears a little too "thin" for me -- mainly the "dot" is indistinct due to anti-aliasing making almost no pixels pure white.

If you set the `strokeWidth` to 2.5, it's just right IMHO--

Current value of 2:
![Screenshot 2023-11-17 at 11 58 41 AM](https://github.com/msupply-foundation/open-msupply/assets/5456533/933f99e3-888c-4ee0-8520-32d5f77f7e5b)

Changed to 2.5:
![Screenshot 2023-11-17 at 11 58 51 AM](https://github.com/msupply-foundation/open-msupply/assets/5456533/54cec3d6-3acc-4c05-ae54-fb732ba151d5)

As I said, super picky 😉

Reply:
🙏  I appreciate the attention to detail! ( it bugged me a little too )